Gemfile.lock

```diff
@@ -468,12 +465,12 @@ GEM
     ttfunk (1.7.0)
     tzinfo (2.0.5)
       concurrent-ruby (~> 1.0)
-    tzinfo-data (1.2022.3)
+    tzinfo-data (1.2022.5)
       tzinfo (>= 1.0.0)
     unf (0.1.4)
       unf_ext
     unf_ext (0.0.8.2)
-    unicode-display_width (2.2.0)
+    unicode-display_width (2.3.0)
```

Comment:
Cool; TIL about this gem. I needed something similar in rex-text a while ago.

That aside the diff looks fine
https://my.diffend.io/gems/unicode-display_width/2.2.0/2.3.0

It seems to be adding unicode 15 support to `data/data/display_width.marshal.gz`

That file is a gzipped binary blob that gets loaded via `Marshal.load` - and there isn't too much context on how to verify that file or where it came from

```ruby
module Unicode
  class DisplayWidth
    File.open(INDEX_FILENAME, "rb") do |file|
      serialized_data = Zlib::GzipReader.new(file).read
      serialized_data.force_encoding Encoding::BINARY
      INDEX = Marshal.load(serialized_data)
    end
  end
end
```

https://github.com/janlelis/unicode-display_width/blob/ef4731c2ed9b1e884d6cff4d3ea12e88a6b8c148/lib/unicode/display_width/index.rb#L6-L14

It's probably fine, but I'll circle back round to this